Tools/CISupport/ews-app/ews/common/buildbot.py

```diff
@@ -169,3 +169,13 @@ def retry_build(cls, builder_id, build_number):
 
         _log.error('Failed to retry build: {}, http response code: {}'.format(build_url, response.status_code))
         return False
+
+    @classmethod
+    def is_tester_queue(self, queue):
+        icon = Buildbot.icons_for_queues_mapping.get(queue)
+        return icon in ['testOnly', 'buildAndTest']
+
+    @classmethod
+    def is_builder_queue(self, queue):
+        icon = Buildbot.icons_for_queues_mapping.get(queue)
```

Comment:
Shouldn't be a blocker for landing, but this looks like a roundabout approach. This makes a model level decision (which queue it is) from presentation data (what icon it has).